package.json

```diff
@@ -3,13 +3,15 @@
   "version": "0.0.0",
   "private": true,
   "scripts": {
-    "test:browser": "playwright test --headed",
-    "test": "playwright test",
+    "test:browser": "playwright test --workers=1 --headed",
+    "test": "playwright test --workers=1",
```

Comment:
We need to remove --workers=1 @sidharthWeb3auth

Reply:
Ideally tests should run in parallel, and the things that we want to run in serial, should be done via test.serial

Reply:
Yeah but the problem is openlogin gets overloaded and blocks with "Too many requests, Please try again in sometime" due to rate limiting. Limiting to one worker will slow down the CI pipe but ensures openlogin wont trigger the rate limiter. @PoojaChopra